src/Redis/RedisSessionHandler.php

```diff
@@ -63,7 +63,7 @@ public function read($sessionId)
 
                 return $this->redis->get($sessionId);
             }
-        } catch (\M6Web\Component\Redis\Exception $e) {
+        } catch (Exception $e) {
```

Comment:
this class is not in the same namespace as this exception

so why did it has to change?

Reply:
🙈 myBad, It's not the same namespace like this bundle 😓 .